src/python/grpcio/tests/protoc_plugin/beta_python_plugin_test.py

```diff
@@ -244,35 +244,63 @@ class PythonPluginTest(unittest.TestCase):
   methods and does not exist for response-streaming methods.
   """
 
-  def setUp(self):
```

Comment:
That was the intention.  The reason I didn't use ```setUpClass``` and ```tearDownClass``` is because they didn't work.  I did a little bit of investigation, and it looks like these methods work when running the test normally, but not when using our ```test_lite``` thing.

I've just reverted to using ```setUp``` and ```tearDown```